core/unit_test/TestBitManipulationBuiltins.hpp

```diff
@@ -789,7 +789,14 @@ struct TestBitCastFunction {
     ASSERT_EQ(errors, 0) << "Failed check no error for bit_cast()";
   }
   template <typename To, typename From>
-  static KOKKOS_FUNCTION bool check(const From& from) {
+#if defined(KOKKOS_COMPILER_GCC) && (910 <= KOKKOS_COMPILER_GCC) && \
+    (KOKKOS_COMPILER_GCC < 930)
```

Comment:
AFAICT `KOKKOS_COMPILER_GCC` doesn't exist.
```suggestion
#if defined(KOKKOS_COMPILER_GNU) && (910 <= KOKKOS_COMPILER_GNU) && \
    (KOKKOS_COMPILER_GNU < 930)
```